CCS2.0/CCS2.0/Views/Home/ViewPost.cshtml

```diff
@@ -310,12 +193,24 @@
                                     </div>
                                     <input type="number" name="Comments.Flag" hidden value="0">
                                     <input type="number" name="Comments.ImageId" hidden value="@Model.ImageId">
+
+                                    <div class="col-md-offset-2 col-md-10">
+                                        <recaptcha />
+                                        <span class="text-danger" id="recaptchaErrorMessage"></span>
+
+                                    </div>
+                                    <br />
                                     <div class="modal-footer" style="border-top:none">
                                         <input type="submit" value="Post Comment" class="btn btn-primary" />
                                         <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
                                     </div>
                                 </div>
                             </form>
+                            @section Scripts {
+                                <recaptcha-script validation-message-element-id="recaptchaErrorMessage" />
+                                <script src="https://www.google.com/recaptcha/api.js" async defer></script>
```

Comment:
Don't you pass in the Sitekey here? Just curious if this is working.

Reply:
the site key is under "manage user secrets". We got it to work, and it's tracking it under the Google reCAPTCHA website.